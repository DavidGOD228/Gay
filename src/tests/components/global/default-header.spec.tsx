import 'react-native'
// Note: test renderer must be required after react-native.
import React from 'react'
import renderer from 'react-test-renderer'
import DefaultHeader from '../../../components/global/default-header'

it('renders correctly', () => {
  const tree = renderer.create(
    <DefaultHeader/>
  )
  expect(tree)
    .toMatchSnapshot()
})
