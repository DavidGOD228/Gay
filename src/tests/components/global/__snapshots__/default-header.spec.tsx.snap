// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <Image
    resizeMode="cover"
    source={1}
    style={
      Object {
        "flex": 1,
        "height": 65,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "flex-end",
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={1}
        />
      </View>
      <View
        style={
          Array [
            Object {
              "height": 10,
              "width": 1,
            },
            Object {
              "height": 10,
            },
          ]
        }
      />
    </View>
  </Image>
</View>
`;
